Src/Support/Google.Apis.Auth.Tests/GoogleJsonWebSignatureTests.cs

```diff
@@ -0,0 +1,155 @@
+﻿/*
+Copyright 2017 Google Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using Google.Apis.Tests.Mocks;
+using System;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Google.Apis.Auth.Tests
+{
+    public class GoogleJsonWebSignatureTests
+    {
+        // From https://www.googleapis.com/oauth2/v3/certs
+        const string GoogleCertsJson = @"
+{
+ ""keys"": [
```

Comment:
There are two reasons it doesn't download from `googleapis.com` during the unit test:
1. This is a unit test, which shouldn't require anything external during runtime.
2. The keys provided at `googleapis.com/oauth2/v3/certs` rotate over a period of days, so the test JWT will no longer be "valid" according to certs downloaded in the future.

There is an integration test that checks the download from `googleapis.com` happens successfully. And I think I'll probably add one more integration test that checks that validation succeeds of a real JWT freshly acquired from Google.